### PR TITLE
Tiny Updates to doc for Value Converters

### PR DESCRIPTION
**Pull request by greystate**
Mostly a lot of "a" that should be "an" :)

### CODE REVIEW

File: Extending/Property-Editors/value-converters.md [hunk 1]
```diff
@@ -2,18 +2,18 @@
...
 Starting with Umbraco v7, published property values have four "Values":
 
-- Data - The raw data stored in the database, this is generally a `String`
-- Source - A object of type that is appropriate to the property, e.g. a nodeId should be a `Int` or a collection of nodeId's would be a array of `Int[]`
-- Object - The object to be used when accessing the property using a Published Content API e.g. UmbracoHelper `GetPropertyValue<T>` method
-- XPath - The object to be used when the property is accessed by XPath, this should generally be a `String` or a `XPathNodeIterator` 
+- **Data** - The raw data stored in the database, this is generally a `String`
+- **Source** - An object of a type that is appropriate to the property, e.g. a nodeId should be an `Int` or a collection of nodeIds would be an integer array, `Int[]`
+- **Object** - The object to be used when accessing the property using a Published Content API, e.g. UmbracoHelper's `GetPropertyValue<T>` method
+- **XPath** - The object to be used when the property is accessed by XPath; This should generally be a `String` or an `XPathNodeIterator`
```
Comment (dampee) [inline Extending/Property-Editors/value-converters.md]: do we need a ;-sign?

File: Extending/Property-Editors/value-converters.md [hunk 2]
```diff
@@ -25,9 +25,9 @@ Implement `IPropertyValueConverter` from the `Umbraco.Core` namespace on your cl
 
 ### bool IsConverter(PublishedPropertyType propertyType) ###
 
-This method is called for each PublishedPropertyType (document type property) at application startup. By returning True your value converter will be registered for that property type and your conversion methods will be executed when ever that value is requested. 
+This method is called for each PublishedPropertyType (document type property) at application startup. By returning `True` your value converter will be registered for that property type and your conversion methods will be executed when ever that value is requested. 
 
-Example, checking if the PublishedPropertyType PropertyEditorAlias property is equal to the alias of the core content editor
+Example: Checking if the PublishedPropertyType PropertyEditorAlias property is equal to the alias of the core content editor
```
Comment (dampee) [inline Extending/Property-Editors/value-converters.md]: I am not native English, but adding a upper case after a : feels odd.

**Comment by dampee** [comment]
Thanks for the review of the page!  I do have a few questions.  Do you have the change to review them?

File: Extending/Property-Editors/value-converters.md (hunk 1 above)
Comment (greystate) [reply Extending/Property-Editors/value-converters.md]: Could easily do with a period (**.**) instead, if the semicolon feels weird :-)

File: Extending/Property-Editors/value-converters.md (hunk 2 above)
Comment (greystate) [reply Extending/Property-Editors/value-converters.md]: To my knowledge (and eyes too), unless what follows the colon is a list of things, the first world should be uppercased as in a new sentence. The way I read it is like this: "Here's an example: Checking if [...]".

Comment (Jeavon) [reply Extending/Property-Editors/value-converters.md]: Well it actually depends on British English or American English, https://www.grammarly.com/blog/capitalization-after-colons/. So as the Umbraco Docs are mainly American English, lets go with the capitalization (see the z)

**Comment by dampee** [comment]
Hi Chriztian / @greystate 
Thanks for your help with the docs!  It is very much appreciated! H5YR

**Pull request merged**